src/main/java/hudson/plugins/ccm/parser/CcmParser.java

```diff
@@ -141,4 +134,43 @@ public CcmParser(String defaultEncoding) {
 		return annotations;
 	}
 
+	private Priority calculateMetricPriority(Metric metric){
+		Priority priority;
+		if (isMetricHighPriority(metric)) {
+			priority = Priority.HIGH;
+		} else if (isMetricModeratePriority(metric)) {
+			priority = Priority.NORMAL;
+		} else {
+			priority = Priority.LOW;
+		}
+
+		return priority;
+	}
+
+	private boolean isMetricHighPriority(Metric metric){
+		String metricClassification = metric.getClassification();
+		if (metricClassification.contains("high")){
+			return true;
+		}
+		else if (metricClassification.contentEquals("C") || metricClassification.contentEquals("D") ||
```

Comment:
Hi, do you have an updated CCM output that emits these C, D, E, F value? The only one that I have (stopped using/building for windows long time ago) is this one - https://github.com/jenkinsci/ccm-plugin/blob/master/src/test/resources/hudson/plugins/ccm/parser/ccm.result.xml